tensorflow/examples/skflow/mnist_weights.py

```diff
@@ -34,8 +35,10 @@
 
 classifier = learn.TensorFlowLinearClassifier(
     n_classes=10, batch_size=100, steps=1000, learning_rate=0.01)
-classifier.fit(mnist.train.images, mnist.train.labels)
-score = metrics.accuracy_score(mnist.test.labels, classifier.predict(mnist.test.images))
+X_train, y_train = mnist.train.images.astype(np.float32), mnist.train.labels.astype(np.int)
```

Comment:
This is an outdated example, it's in line to be updated, just has to wait its turn. If the new code cannot handle uint8s it should.